java/test/org/openqa/selenium/grid/router/StressTest.java

```diff
@@ -124,4 +132,15 @@ void multipleSimultaneousSessions() throws Exception {
 
     CompletableFuture.allOf(futures).get(4, MINUTES);
   }
+
+  @Test
+  void testStopTimedOutSession() throws Exception {
+    assertThat(server.isStarted()).isTrue();
+    WebDriver driver =
```

Comment:
Perhaps I am too picky, but this class has a specific purpose (Stress Test). To make sense, we either rename the class to something different or move the new test to its class.

Reply:
I updated it to `multipleSimultaneousSessionsTimedOut()`